### ISSUE TITLE
error in sync project

### ISSUE THREAD

**Issue by Rezalatifi** [Horizontal-org/Tella-Android#313]
## Steps to reproduce the problem

1.why failed resolve: com.simplify:ink:1.0.2 in sync this project?? Thank you
1.
1.

**Comment by ahlem-jarrar** [comment]
Can you please provide us your Android Studio version and Gradle version?

**Comment by n8fr8** [comment]
We are experiencing the same issue, and others as well, when trying to do a clean checkout and build.

A number of libraries are no longer available as they were on jcenter which has been shutdown.

We have been able to make some small changes to fix these, finding alternate releases that are compatible, including:

diff --git a/build.gradle b/build.gradle
index 72adbe1a..8db12e66 100644
--- a/build.gradle
+++ b/build.gradle
@@ -55,7 +55,7 @@ buildscript {
     versions.cameraview = '2.7.2'
     versions.exifinterface = '1.3.3'
     versions.simplifyInk = '1.0.2'
-    versions.circularimageview = '3.2.0'
+    versions.circularimageview = '3.0.2'
     versions.ihsanbalLoggingInterceptor = '3.0.0'
     versions.cardview = '1.0.0'
     versions.buildGradle = '4.1.3'
diff --git a/mobile/build.gradle b/mobile/build.gradle
index 3a990874..656676d6 100644
--- a/mobile/build.gradle
+++ b/mobile/build.gradle
@@ -131,7 +131,7 @@ dependencies {
     implementation "androidx.navigation:navigation-runtime-ktx:$versions.navigationRuntime"
     implementation "androidx.navigation:navigation-fragment-ktx:$versions.navigationFragment"
     implementation "androidx.databinding:databinding-runtime:$versions.databindingRuntime"
-    kapt("org.permissionsdispatcher:permissionsdispatcher-processor:$versions.permissionDispatcher")
+    kapt("com.github.permissions-dispatcher:permissionsdispatcher-processor:$versions.permissionDispatcher")
 
     implementation "com.squareup.okhttp3:logging-interceptor:$versions.okhttp3LoggingInterceptor"
     implementation "com.squareup:tape:$versions.squareupTape"
@@ -172,7 +172,8 @@ dependencies {
     kapt "androidx.hilt:hilt-compiler:$versions.hilt_version"
     //kapt "org.jetbrains.kotlinx:kotlinx-metadata-jvm:0.6.0"
 
-    implementation "com.simplify:ink:$versions.simplifyInk"
+    //implementation "com.simplify:ink:$versions.simplifyInk"
+    implementation "com.github.nak5ive:ink-android:1.0.3"
     implementation "com.mikhaellopez:circularimageview:$versions.circularimageview"
     implementation "org.apache.directory.studio:org.apache.commons.io:$versions.apacheCommonsIo"
     implementation "com.github.ihsanbal:LoggingInterceptor:$versions.ihsanbalLoggingInterceptor"

**Comment by n8fr8** [comment]
com.github.nak5ive:ink-android:1.0.3 is a fork of the original simplify ink library

com.github.permissions-dispatcher is a version of the org.permissionsdispatcher that is available via jitpack

**Comment by n8fr8** [comment]
There also seem to be missing resources:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':shared-ui:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/Volumes/minimi/dev/repos/Tella-Android/shared-ui/build/intermediates/merged_res/release/values/values.xml:4876: AAPT: error: resource drawable/rounded_light_button_background (aka org.hzontal.shared_ui:drawable/rounded_light_button_background) not found.
         
     ERROR:/Volumes/minimi/dev/repos/Tella-Android/shared-ui/build/intermediates/merged_res/release/values/values.xml:4877: AAPT: error: resource color/wa_dark_gray (aka org.hzontal.shared_ui:color/wa_dark_gray) not found.
         
     ERROR:/Volumes/minimi/dev/repos/Tella-Android/shared-ui/build/intermediates/merged_res/release/values/values.xml:4909: AAPT: error: resource dimen/default_edittext_layout_height (aka org.hzontal.shared_ui:dimen/default_edittext_layout_height) not found.

**Comment by ahlem-jarrar** [comment]
Thank you, @Rezalatifi! I just saw the fork, and it looks like the problem has been resolved on your end.

**Comment by n8fr8** [comment]
> @n8fr8 Perhaps you could reach out to us for a call if the issue isn't fully resolved, or if you have any further questions?

Okay. Have you made any changes in your develop or main branches to address this? Can you reproduce the problem on your side?

**Comment by ahlem-jarrar** [comment]
@n8fr8 yes we were able to reproduce the problem we will push the changes on both master and develop.

**Comment by ahlem-jarrar** [comment]
@n8fr8 This PR aims to resolve all related build issues. Please ensure you generate your own google-services.json file and add it to the project for successful integration. The PR is currently under review and is expected to be merged by tomorrow. Feel free to reach out if you need any further assistance.

[Link to PR](https://github.com/Horizontal-org/Tella-Android/pull/325)
